CHANGELOG.md

```diff
@@ -32,6 +32,14 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 - Restrict IPv6 access to IMDS to root and cluster admin users only, when configuration parameter `HeadNode/Imds/Secured` is enabled.
 - Set Slurm configuration `AuthInfo=cred_expire=70` to reduce the time requeued jobs must wait before starting again when nodes are not available.
 - Move `SelectTypeParameters` and `ConstrainRAMSpace` to the `parallelcluster_slurm*.conf` include files.
+- Upgrade third party cookbooks.
```

Comment:
can you write the changelog similar to the previous update, e.g.
```
Upgrade third-party cookbook dependencies:
  apt-7.4.0 (from apt-7.3.0)
  iptables-8.0.0 (from iptables-7.1.0)
  line-4.0.1 (from line-2.9.0)
  openssh-2.9.1 (from openssh-2.8.1)
  pyenv-3.4.2 (from pyenv-3.1.1)
  selinux-3.1.1 (from selinux-2.1.1)
  ulimit-1.1.1 (from ulimit-1.0.0)
  yum-6.1.1 (from yum-5.1.0)
  yum-epel-4.1.2 (from yum-epel-3.3.0)
```

Reply:
Sure, will do. Thanks